driver-portal/openshift/templates/driver-portal.template.yml

```diff
@@ -0,0 +1,326 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: driver-portal-template
+
+parameters:
+  - description: openshift namespace prefix
+    name: NAMESPACE_PREFIX
+    required: true
+  - description: openshift namespace env name
+    name: ENV
+    required: true
+  - description: app name
+    name: APP_NAME
+    required: true
+  - description: image tag for deployments
+    name: TAG
+    required: true
+  - description: host for routes
+    name: HOST
+    required: true
+  - description: ASPNET runtime environment (Development/Staging/Production)
+    name: RUNTIME_ENVIRONMENT
+    value: Staging
+  - description: oauth server url
+    name: OAUTH_SERVER_URL
+    required: false
+  - description: oauth server client id
+    name: OAUTH_CLIENTID
+    required: false
+  - description: oauth server client secret
+    name: OAUTH_CLIENTSECRET
+    required: false
+  - description: oidc client id
+    name: OIDC_CLIENTID
+    required: false
+  - description: oidc scopes
+    name: OIDC_SCOPE
+    value: "openid profile email driver-portal-api"
+  - description: CORS_ORIGINS
+    name: CORS_ORIGINS
+    value: "true"
+
+objects:
+  - kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: ${APP_NAME}-driver-portal-api-config
+      namespace: ${NAMESPACE_PREFIX}-${ENV}
+      labels:
+        app: ${APP_NAME}-driver-portal
+        app.kubernetes.io/part-of: ${APP_NAME}
+    data:
+      ASPNETCORE_ENVIRONMENT: ${RUNTIME_ENVIRONMENT}
+      KEY_RING_PATH: "/dpkeys"
+      app__cors__origins: ${CORS_ORIGINS}
+  - kind: Secret
+    apiVersion: v1
+    type: Opaque
+    metadata:
+      name: ${APP_NAME}-driver-portal-api-secrets
+      namespace: ${NAMESPACE_PREFIX}-${ENV}
+      labels:
+        app: ${APP_NAME}-driver-portal
+        app.kubernetes.io/part-of: ${APP_NAME}
+    stringData:      
+      DATABASE_LOCATION: '/db/DATA.db'
```

Comment:
this is not relevant

Reply:
Deleted all the references to DB in the openshift template